frontend/public/components/storage/access-modes/access-mode.tsx

```diff
@@ -1,15 +1,11 @@
+import * as _ from 'lodash';
```

Comment:
Can you explain why this was moved to the public folder? We're actually trying to migrate the components the other way (from public to console-shared or console-app).

Reply:
These two files can reused by other if it is in public, If it is not correct, can we move create-pvc also somewhere else with these two files?

Reply:
console-shared is the right package for reusable components.